wallets/wallet-faq.md

```diff
@@ -4,26 +4,26 @@ description: All questions surrounding wallets
 
 # Wallet FAQ
 
-## **Which wallet\(s\) can I use for AR tokens?**
+## **Which wallet(s) can I use for AR tokens?**
 
-AR is currently only compatible with the Arweave web extension wallet. You can download it  from [**Chrome**](https://chrome.google.com/webstore/detail/arweave/iplppiggblloelhoglpmkmbinggcaaoc), [**Firefox**](https://addons.mozilla.org/en-US/firefox/addon/arweave/), or [**Brave**](https://chrome.google.com/webstore/detail/arweave/iplppiggblloelhoglpmkmbinggcaaoc).
+The following are wallets (developed by the Arweave community)  which are capable of storing AR tokens.
 
-## **How do I use the Arweave web extension wallet?**
+<table><thead><tr><th> </th><th data-type="checkbox">arweave.app</th><th data-type="checkbox">ArConnect</th><th data-type="checkbox">Finnie</th></tr></thead><tbody><tr><td>AR tokens</td><td>true</td><td>true</td><td>true</td></tr><tr><td>dApp permissions</td><td>true</td><td>true</td><td>true</td></tr><tr><td>Open Source</td><td>true</td><td>true</td><td>false</td></tr><tr><td>No Install</td><td>true</td><td>false</td><td>false</td></tr><tr><td>NFTs</td><td>false</td><td>true</td><td>true</td></tr><tr><td>PSTs</td><td>false</td><td>true</td><td>false</td></tr><tr><td>Page Archiving</td><td>false</td><td>true</td><td>true</td></tr></tbody></table>
+
+## **How do I use the arweave.app web wallet?**
 
 Lucky for you, we’ve created a step-by-step guide which can be [**found here**](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet)**.**
 
 ## **Can I use the Arweave wallet as a cold wallet?**
```

Comment:
Arweave => arweave.app

Reply:
![image](https://user-images.githubusercontent.com/3269261/160461705-902c2133-7c62-4b35-bbe0-37a455336d59.png)
How things look in gitbook ;)